mcs/class/System.ServiceModel.Syndication/Makefile

```diff
@@ -4,7 +4,11 @@ include ../../build/rules.make
 
```

Comment:
I just noticed we don't build the `mcs/class/System.ServiceModel.Syndication` folder at all in any profile so we can just kill the entire folder 😏

Reply:
well, let's try :)